package.json

```diff
@@ -1,13 +1,24 @@
 {
   "name": "react-ravepayment",
-  "version": "2.0.8",
+  "version": "3.0.0",
   "description": "This is a reactJS library for implementing rave payment gateway",
-  "main": "distribution/RavePayment.min.js",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "typings": "dist/index.d.ts",
+  "jsnext:main": "dist/index.es.js",
   "scripts": {
-    "test": "mocha --reporter spec test/testSetup.js \"test/**/*.spec.js\"",
-    "start": "react-scripts start",
+    "prepublish": "npm run build",
+    "prepare": "npm run build",
```

Comment:
script on L10 and L11 are doing the same thing. Or am i seeing something different?

Reply:
Ohhh yeah I think they do

Reply:
I set it up so that when there is a need to deploy the docs into github pages or netlify it would be easy to just use any of the script I will remove one of them